src/HotChocolate/AspNetCore/src/AspNetCore/Extensions/EndpointRouteBuilderExtensions.cs

```diff
@@ -141,15 +147,15 @@ public static class EndpointRouteBuilderExtensions
     /// </exception>
     public static GraphQLHttpEndpointConventionBuilder MapGraphQLHttp(
         this IEndpointRouteBuilder endpointRouteBuilder,
-        RoutePattern? pattern = default,
+        RoutePattern pattern,
         NameString schemaName = default)
     {
         if (endpointRouteBuilder is null)
         {
             throw new ArgumentNullException(nameof(endpointRouteBuilder));
         }
 
-        pattern ??= Parse("/graphql");
+        pattern ??= Parse(GraphQLHttpPath);
```

Comment:
since we have the string overload with default this now has become not necessary anymore. RoutePattern from the signature is no longer allowed to be null so we should throw an ArgumentException if null is passed.

Reply:
Done